pages/blog/getting-the-best-out-of-your-graphql-client-by-using-relay-patterns-through-graphql-code-generator.mdx

```diff
@@ -0,0 +1,212 @@
+import withArticle from '../../ui/blog/article'
+
+export const meta = {
+  title:
+    'Getting the best out of your GraphQL client by using Relay patterns through GraphQL Code Generator',
+  author: 'arda',
+  tags: ['graphql'],
+  date: '2022-08-01',
+  description: 'TBA',
+  image:
+    '/blog-assets/getting-the-best-out-of-your-graphql-client-by-using-relay-patterns-through-graphql-code-generator/cover.jpg'
+}
+
+export default withArticle(meta)
+
+# Getting the best out of your GraphQL client by using Relay patterns through GraphQL Code Generator
+
+There is no doubt that Relay is the most advanced JavaScript GraphQL Client that is out there. However, at the same time, the learning curve and adoption is yet very sparse compared to other popular alternatives such as Apollo Client, Urql, or GraphQL Request.
+
+As a consequence, many people don’t even know about all the benefits and patterns used within (and so far being exclusive to ) the relay-compiler, the GraphQL Code Generator counter-part tailored for Relay applications.
+
+The Guild gathered and worked on projects whose GraphQL client usage spans across all the available solutions today. In our opinion, the most important parts of Relay are the concepts of building and scaling applications. These patterns are actually applicable to any GraphQL Client, you just need the right code generation tool for the job.
+
```

Comment:
Now here needs to come an overview of these concepts and why they are beneficial

Reply:
including diagrams, you can also copy these things from the first ever GraphQL talk or some the Relay website